server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,116 @@
+[id="proc-creating-oidc-client_{context}"]
+==== Creating an Open ID Connect Client
+
+. Click the `Clients` left menu item.  
+. Click the `Create` button. This will bring you to the `Add Client` page.
+
+.Clients
```

Comment:
We need a + (plus sign) before the image when the procedure is continuing.

See this file for a sample of how to format it: https://raw.githubusercontent.com/andymunro/keycloak-documentation/master/getting_started/topics/first-realm/proc-create-realm.adoc